docs/basic-features/data-fetching/incremental-static-regeneration.md

```diff
@@ -88,6 +88,8 @@ When a request is made to a page that was pre-rendered at build time, it will in
 
 When a request is made to a path that hasn’t been generated, Next.js will server-render the page on the first request. Future requests will serve the static file from the cache. ISR on Vercel [persists the cache globally and handles rollbacks](https://vercel.com/docs/concepts/next.js/incremental-static-regeneration).
 
+> Note:  Check if your upstream data provider has caching enabled by default. You might need to disable (e.g. `useCdn: false`), otherwise a revalidation won't be able to pull fresh data to update the Next.js cache. When using Next.js on Vercel, caching is enabled by default through the Vercel Edge Network.
```

Comment:
Slightly modified - thoughts?

Reply:
> When using Next.js on Vercel, caching is enabled by default through the Vercel Edge Network

Is this referring to API routes, as caching isn't enabled by default there unless the `cache-control` header is set. Also not sure how common it is to fetch from a Vercel endpoint during a revalidation instead of database or CMS like Contentful

Reply:
Tweaked about more in https://github.com/vercel/next.js/pull/38014/commits/bead242ff01a5ee0e3789a0d216eadc56b63d647 wdyt?